installer/templates/phx_single/gitignore

```diff
@@ -32,3 +32,6 @@ npm-debug.log
 # we ignore priv/static. You may want to comment
 # this depending on your deployment strategy.
 /priv/static/
+
+# Temporary files for e.g. tests
```

Comment:
In Elixir core, we removed the use of Latin abbreviations to simplify the reading. I think in this case we used "for example"

Reply:
This is a 1:1 copy of what is in elixir core though.

Reply:
> This is a 1:1 copy of what is in elixir core though.

Then it should be fixed

Reply:
I think "the agreement" was to do it with all Latin expressions, but we only implemented it for "etc."
The discussion is here: https://github.com/elixir-lang/elixir/pull/9559

Reply:
Seems like it was indeed fixed in elixir core, so I copied the wording from there